sjb/config/test_cases/test_branch_origin_service_catalog.yml

```diff
@@ -10,6 +10,13 @@ extensions:
     - name: "origin"
     - name: "service-catalog"
   actions:
+    - type: "script"
```

Comment:
You should migrate from `sync_repos:` to `sync:` and this will happen for you automatically

Reply:
@stevekuznetsov will this work for both PRs and post submits?
```
sync:
- "openshift,service-catalog=${PULL_BASE_REF}"
```
If so, I can fix that up.

Reply:
That should work, yes

Reply:
Actually, the syntax document says sync and sync_repos can't be mixed. That would mean that common/test_cases/origin.yml would need migrating first, right?